src/client/components/MethodologyPage/MethodologyBanner/MethodologyBanner.tsx

```diff
@@ -0,0 +1,56 @@
+import type { FunctionComponent } from 'react';
+import React, { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+import logger from '../../../../server/logger';
+import Button, { ButtonSize } from '../../Button/Button';
+import styles from './MethodologyBanner.scss';
+
+const MethodologyBanner: FunctionComponent = () => {
+  const navigate = useNavigate();
+  const getAQuoteClick = useCallback(() => {
+    try {
+      navigate('./contact-us');
```

Comment:
Once I add the "link" button, we should use that here instead of making our own callback for onClick.